### DISCUSSION TITLE
Foreign Key constraint failed - DB Record Creation

### DISCUSSION THREAD

**Discussion by josephbobby24** [prisma/prisma#15568] [Q&A] [answered]
Models: 
`model Profile {
  userId     String      @unique
  Freelancer Freelancer? @relation(fields: [userId], references: [userId], onDelete: Cascade)
}

model Freelancer {
  userId    String   @unique
  portfolio String   @default("No portfolio!")
  timezone  String?
  paypal    String   @default("No paypal set!")
  Profile   Profile?
}
`

Code:
`const dbProfile = await this.client.db.profile.create({
            data: data
        });`

data is just { userId: "TestUser" }

Error:
`const dbProfile = await this.client.db.profile.create(
Foreign key constraint failed on the field: `userId`
    at RequestHandler.handleRequestError (C:\Users\josep\OneDrive\Desktop\Bots\Navi\node_modules\@prisma\client\runtime\index.js:29909:13)`

**Comment by josephbobby24** [comment, marked as answer]
Nevermind! I fixed it!
Apologies.

Just did the references on the freelancer model instead.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
